deploy/helm/cron-set-controller/templates/crds.yaml

```diff
@@ -0,0 +1,9481 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
um. In the following the helm docs, basically using special directory called crds.
https://helm.sh/docs/chart_best_practices/custom_resource_definitions/

Is there any different with your method?

Reply:
I missed this. I'll fix it.